content/en/apps/reference/targets.md

```diff
@@ -20,30 +20,30 @@ All targets are defined in the `targets.js` file as an array of objects accordin
-|---|---|---|---|
-| `id` | `string` | An identifier for the target. | yes, unique |
-| `icon` | `string` | The icon to show alongside the target. Should correspond with a value defined in `resources.json`. | no |
-| `translation_key` | `translation key` | Translation key for the title of this target. | no, but recommended |
-| `subtitle_translation_key` | `translation key` | Translation key for the subtitle of this target. If none supplied the subtitle will be blank. | no |
-| `percentage_count_translation_key` | `translation key` | Translation key for the percentage value detail shown at the bottom of the target, eg "(5 of 6 deliveries)". The translation context has `pass` and `total` variables available. If none supplied this defaults to `targets.count.default`. | no |
-| `context` | `string` | A string containing a JavaScript expression. This widget will only be shown if the expression evaluates to true. Details of the current user is available through the variable `user`. | no |
-| `type` | `'count'` or `'percent'` | The type of the widget. | yes |
-| `goal` | `integer` | For targets with `type: 'percent'`, an integer from 0 to 100. For `type: 'count'`, any positive number. If there is no goal, put -1. | yes |
-| `appliesTo` | `'contacts'` or `'reports'` | Do you want to count reports or contacts? This attribute controls the behavior of other attributes herein. | yes |
-| `appliesToType` | If `appliesTo: 'reports'`, an array of form codes. If `appliesTo: 'contacts'`, an array of contact types. | Filters the contacts or reports for which `appliesIf` will be evaluated. For example, `['person']` or `['clinic', 'health_center']`. For example, `['pregnancy']` or `['P', 'pregnancy']`. | no |
-| `appliesIf` | `function(contact, report)` | If `appliesTo: 'contacts'`, this function is invoked once per contact and `report` is undefined. If `appliesTo: 'reports'`, this function is invoked once per report. Return true to count this document. For `type: 'percent'`, this controls the denominator. | no |
-| `passesIf` | `function(contact, report)` | For `type: 'percent'`, return true to increment the numerator. | yes, if `type: 'percent'`. Forbidden when `groupBy` is defined. |
-| `date` | `'reported'` or `'now'` or `function(contact, report)` | When `'reported'`, the target will count documents with a `reported_date` within the current month. When `'now'`, target includes all documents. A function can be used to indicate when the document should be counted. Default is `'reported'`. | no |
+| property | type | description                                                                                                                                                                                                                                                                                                                                                                                                                   | required |
```

Comment:
Please remove the other changes, I mean the weird spaces, it makes the code hard to read. 
![Screen Shot 2022-12-05 at 1 10 47 pm](https://user-images.githubusercontent.com/66472237/205561902-a796ab12-2185-404d-b6cf-0a5ba339b02e.png)

Please just leave the relevant change for this ticket.

Reply:
@latin-panda yes I have a PR with the changes on task-owner vs requester I am waiting to merge the changes

Reply:
Awesome thanks @esthermmoturi! 🤩

Reply:
Yes thanks Esther, about the weird spaces, it is something that my IDE was doing when showing the preview besides, I managed to fix, thanks for noticing.